docs/IR.md

```diff
@@ -141,13 +131,13 @@ ir_version|int32|The ONNX version corresponding to the operators.
 ir_version_prerelease|string|The prerelease component of the SemVer of the IR.
 ir_build_metadata|string|The build metadata of this version of the operator set.
 domain|string|The domain of the operator set. Must be unique among all sets.
-opset_version|int64|The version of the set of operators.
-doc_string|string|A human-readable documentation for this set of operators. Markdown is allowed.
-operator|Operator[]|The operators of this operator set.
+opset_version|int64|The version of the operator set.
+doc_string|string|Human-readable documentation for this operator set. Markdown is allowed.
+operator|Operator[]|The operators contained in this operator set.
 
-The operator set version is a simple integer value that is monotonically increased as new versions of the operator set are published.
+The operator set version is a simple monotonically increasing integer.
```

Comment:
The wording here is a bit confusing in the formal sense (though understandable). The earlier explanatory test seems to clarify the meaning of "increasing" better.